km_api/know_me/admin.py

```diff
@@ -39,6 +39,28 @@ class EmergencyItemAdmin(admin.ModelAdmin):
     search_fields = ('name',)
 
 
+@admin.register(models.EmergencyContact)
+class EmergencyContactAdmin(admin.ModelAdmin):
+    """
+    Admin for the ``EmergencyContact`` model.
+    """
+    fields = ('name', 'relation', 'phone_number', 'alt_phone_number', 'email')
+    list_display = ('name', 'km_user')
+    search_fields = ('name',)
+
+
+@admin.register(models.ImageContent)
+class ImageContentAdmin(admin.ModelAdmin):
```

Comment:
I believe we removed the `ImageContentAdmin` in favor of the `ImageContentInline` at the top of the file. If I had to guess I would say this snuck back in during a rebase.